irodsConnector/manager.py

```diff
@@ -113,9 +109,7 @@ def irods_env_file(self, filepath: str):
 
         """
         self._irods_env_file = filepath
-        logging.debug(f'setting: {self._irods_env_file=}')
-        if self.session:
-            self.session.irods_env_file = filepath
+        logging.info(f'setting: {self._irods_env_file=}')
```

Comment:
Unless you need to know the value of the variable, it is better to indicate only that the variable is set.  If you need to know the value, using `logging.debug` with %-formatting will be more efficient when properties like this are called frequently:

```python
logging.debug('setting: %s', self._irods_env_file)
```

See PR #97 for more examples.  I can also update this there while I'm doing the rest.

Reply:
This and L138 really should be _debug_ level.  The user normally does not need this information unless something unusual is happening.  That is when _debug_ is used.